spanner_orm/api.py

```diff
@@ -39,13 +42,16 @@ def run_read_only(cls, method, *args, **kwargs):
   @staticmethod
   def find(transaction, table_name, columns, keyset):
     """Obtains rows with primary_keys from the given table."""
+    _logger.debug('Find table=%s columns=%s keys=%s',
```

Comment:
the string interpolation format currently being used in this project is:
```python
'{value}'.format(value=<value>)
```

Reply:
logging methods are exception to that to avoid constructing the string if it is not going to be printed(level criteria not met, etc)